client/dfclient/src/menu_state.cpp

```diff
@@ -14,40 +18,119 @@
 #include "menu_state.hpp"
 #include "resources.hpp"
 #include "util.hpp"
+#include "http_client.hpp"
 
 namespace nc = ncine;
 
 nctl::UniquePtr<nc::Sprite> logoSprite;
 nc::Colorf bgColor(0.96875f, 0.97265625, 0.953125, 1.0f);
 
+void MenuState::ShowMessage(std::string message)
+{
+	nc::SceneNode &rootNode = nc::theApplication().rootNode();
+	auto res = nc::theApplication().appConfiguration().resolution;
+	auto text = new ncine::TextNode(&rootNode, _resources.fonts["comic"].get());
+	text->setString(message.c_str());
+	text->setPosition(res.x * 0.5f, res.y * 0.30f);
+	text->setScale(0.5f);
+	text->setColor(0, 0, 0, 255);
+	_resources._tweens.push_back(CreateTextTween(text));
+}
```

Comment:
I still don't like the idea of leaking TextNodes just to show them xd I know it's copied, just nitpicking. Maybe I'll refactor it someday. :D

Reply:
True, I don't know why did I commit this, sorry. Added deleting the text pointer when the tween reaches 0